### ISSUE TITLE
blick.ch: detection

### ISSUE THREAD

**Issue by Flauschibuesi** [uBlockOrigin/uAssets#24977]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://www.blick.ch/`

### Category

detection

### Description

Site detects uBlock since today and blocks usage with an overlay that tells you to disable your adblockers

### Other extensions used

none

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![Screenshot 2024-08-20 212059](https://github.com/user-attachments/assets/0968ef4c-b4bd-491b-a54f-a74a37e4c625)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.59.0
Chromium: 127
filterset (summary):
 network: 155596
 cosmetic: 53180
 scriptlet: 22153
 html: 0
listset (total-discarded, last-updated):
 default:
  user-filters: 0-0, never
  DEU-0: 7541-1, 39m
  easylist: 85374-578, 1m Δ
  easyprivacy: 53047-655, 1m Δ
  plowe-0: 3547-0, now
  ublock-badware: 10514-6, 1m Δ
  ublock-filters: 39780-300, 1m Δ
  ublock-privacy: 1068-2, 1m Δ
  ublock-quick-fixes: 165-0, 1m Δ
  ublock-unbreak: 2497-2, 1m Δ
  urlhaus-1: 29100-0, 39m
filterset (user): [empty]
trustedset:
 added: [array of 7 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 503 ms (selfie)
 maxAssetCacheWait: 287 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 8
 network:
  cxense.com: 2
  googletagmanager.com: 1
  onesignal.com: 1
  ringier-advertising.ch: 1
  sentry.io: 3
 extended:
  ##a[href^="https://jackpot.onelink.me/"]
  ##.jw-wrapper > div[style="opacity: 0; visibility: hidden; overf…
  ##+js(abort-on-property-read, _sp_._networkListenerData)
  ##+js(set-constant, Object.prototype.isAdDisabled, true)
```
</details>

**Comment by Flauschibuesi** [comment]
Sorry to reopen this -  same Problem is back again :/

**Comment by JobcenterTycoon** [comment]
Will check on the next day when no other check before me. Tested on mobile no anti adblock so far.

**Comment by D4niloMR** [comment]
The anti-adblock is coming from `https://cdn.tinypass.com/api/tinypass.min.js`. Also there is no `setNptTechAdblockerCookie` on `view-source:https://www.blick.ch/`

The `rmnt` filter needs `.stay, 1` on my end but it doesn't work even after adding.

Easiest fix is add 

```
||cdn.tinypass.com/api/tinypass.min.js$script,domain=blick.ch
```

**Comment by stephenhawk8054** [comment]
Blocking that will affect the paywall registration at `https://www.blick.ch/blickplus/abos/`

**Comment by stephenhawk8054** [comment]
It's because they are checking `##+js(set, Object.prototype.isAdDisabled, true)` with

```js
if ({}.isAdDisabled) {
  setNptTechAdblockerCookie(true);
  return;
}
```

**Comment by JobcenterTycoon** [comment]
So just

```
blick.ch#@#+js(set, Object.prototype.isAdDisabled, true)
blick.ch##[class*="_ad_"]
blick.ch##aside:has(> div[id^="ad-placement"])
```
? I don’t get video ads with this.

**Comment by stephenhawk8054** [comment]
Yeah, if the old issues are not there, we can switch to cosmetic filters.

There are some other backup filters here https://github.com/uBlockOrigin/uBlock-discussions/discussions/897#discussioncomment-10621706 but we don't need to use currently.
